tests/functional_tests.sh

```diff
@@ -25,15 +25,15 @@ if [ -z "${SUBSCRIPTION_ID}" ] ; then
 fi
 
 if [ ! -f "$PATH_TO_PUBLIC_SSH_KEY" ]; then
-    ssh-keygen -t rsa -b 4096 -f $PATH_TO_PUBLIC_SSH_KEY -N ""
+    ssh-keygen -t rsa -b 4096 -f "$PATH_TO_PRIVATE_SSH_KEY" -N ""
     echo "SSH key created."
 else
     echo "SSH key already exists."
 fi
 
 # Log into Azure (this will open a browser window prompting you to log in)
 echo "Logging you into Azure"
-az login
+#az login
```

Comment:
curious, why is `az login` commented out?

Reply:
Oh woops, I didn't mean to check that in. `az login` always tries to log you in even if you're logged in so I commented this out to test things and was going to follow up with an `if az account show...` type deal later.

Dropped it.